features/other_b/skin.feature

```diff
@@ -41,8 +45,10 @@ Feature: creating and editing skins
    When I am logged in as "skinner"
     And I follow "public skin"
    Then I should see "The skin public skin has been set. This will last for your current session."
-
-
+   When I am logged in as an admin
+     And I follow "Approved Skins"
+     And I check "make_unin_chooser_public_skin"
```

Comment:
Also here -- can you tell me what the three skins appearing on this page are named?